libs/packages/components/src/lib/actions-menu/actions-menu.component.ts

```diff
@@ -19,6 +19,7 @@ export class SdsActionsMenuComponent {
         if (!menuItem.mode) menuItem.mode = this.actionModes.SHOWN;
       });
     }
+    this.model.isIconMenu = this.model.label ? false : true;
```

Comment:
Is the intention is to allow devs to set this field? If so, I think this value should only be set like this if this value is not set. Otherwise there could be a scenario where the expected value gets changed by this line. If that is not the intent, this should probably be made to an internal variable.

Reply:
Removed and refactored the method more generic and added story book documentation